helm-charts/strimzi-kafka-operator/templates/040-Crd-kafka.yaml

```diff
@@ -841,6 +841,8 @@ spec:
                       pattern: '[0-9]+[mMgG]?'
                     gcLoggingEnabled:
                       type: boolean
+                    javaSystemProperties:
+                      type: object
```

Comment:
Is this really the json schema we want? At the json level values are not constrained to strings, so I suspect it leaves us vulnerable to either a deserialization exception or CCE. And it's not self describing. Perhaps it would be better to use a `List<SystemProperty>` at the Java level, so have a self-describing json schema here?